spring-cloud-stream-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -492,4 +494,146 @@ Then do this:
+generated apps have `spring-boot-starter` as the parent. Where is the mismatch? +
+There is no mismatch per se, but a slight subtlety. As the question frames, each app starter has access to artifacts managed all the way through `spring-cloud-build` at compile time.
+However, this is not the case for the generated apps at runtime. Their parent is `spring-boot-starter` that imports `spring-boot-dependencies` bom that includes a majority of the components that these apps need.
+The question then is what if the starter used an artifact managed through the `app-starter-build` or `spring-cloud-build`. Remember that the
+`core-app-starter-dependencies` bom is also included in the generated apps. This bom also imports the `spring-cloud-dependencies` bom. This way, at runtime
+the apps have access to everything that it used at compile time.
+
+.  Why is there an app starter specific BOM in each app starer repositories? For example, `time-app-dependencies`. +
+This is an important BOM. At runtime, the generated apps get the versions used in their dependencies through a BOM that is manaing the dependencies. Since all the boms
+that we specified above only for the helper artifacts, we need a place to manage the starters themselves. This is where the app specific BOM comes into play.
+In addition to this need, as it becomes clear below, there are other uses for this BOM such as dependency overrides etc. But in a nutshell, all the starter dependencies go to this BOM.
+For instance, take TCP repo as an example. It has a starter for source, sink, client processor etc. All these dependencies are managed through the app specific `tcp-app-dependencies` bom.
+This bom is provided to the app generator maven plugin in addition to the core bom. This app specific bom has `spring-cloud-dependencies-parent` as parent.
+
+.  How do I create a new app starter project? +
+If you have a general purpose starter that can be provided as an of of the box app, create an issue for that in https://github.com/spring-cloud-stream-app-starters/app-starters-release[app-starters-release].
+If there is a consensus, then a repository can be created in the `spring-cloud-stream-app-starters` organization where you can start contributing the starters and other components.
+
+.  I created a new starter according to the guidelines above, now how do I generate binder specific apps for the new starters? +
+By default, the app-starters-build in core is configured with the common configuration needed for the app generator maven plugin. It is configured for generating apps for kafka-09, kafka-10 and rabbitmq binders.
+In your starter you already have the configuration specified for the plugin from the parent. Override the names accordingly. Refer to an exsiting starter for guidelines.
+If your app is straightforward, all you have to change is the name of the apps in the <generatedApps> section, and the app-starter specific BOM maven coordinates.
```

Comment:
What do you mean `if your app is straightforward`?  What does changing the name of the apps do wrt to the binder.  This sentence doesn't seem related to the question.